.travis.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
Shall we change it to 2018?

Reply:
I wouldn't, let me just add a new line below